.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+
+---
+
+<!--
+
+We use this repository only for bug reports, if you want to open a feature request,
+please do so on the following repository: https://github.com/RocketChat/feature-requests
+
+If there is a feature request already opened make sure to give your thumps up, as we
```

Comment:
Feature request already opened (where?).   

Are you referring to this repo - and suggesting that we should continue to use this repository for Feature Requests already "gotten in"?   

Please either remove this or clarify this statement.

Reply:
I've said about the other repo on the previous line:

> if you want to open a feature request, please do so on the following repository: https://github.com/RocketChat/feature-requests

maybe having both sentences on the same paragraph helps?

Reply:
what do you think @Sing-Li ?